cogs/example.py

```diff
@@ -16,6 +16,7 @@ async def test(self, ctx, num: int):
         The main command which handles the code execution process.
         """
         await ctx.send(f"The number entered {num}")
+        await ctx.send(f"The number entered {num}+1")
```

Comment:
Should be in the expression because currently it is a string.
E.g {await ctx.send(f"The number entered {num+1}")}